.github/workflows/macos.yml

```
@@ -71,9 +84,36 @@ jobs:
 
       - run: make
 
+      # Launchable requires Python and Java
+      # https://www.launchableinc.com/docs/resources/cli-reference/
+      - name: Set up Python
+        uses: actions/setup-python@871daa956ca9ea99f3c3e30acb424b7960676734 # v5.0.0
+        with:
+          python-version: "3.10"
+      - name: Set up JDK 17
```

I would suggest the following text. We are already using the word "Java" in the "Launchable requires Python and Java" above, and we are also  using the `Set up [Language name]` in the "Set up Python" above.

```suggestion
      - name: Set up Java
```

---

Thanks. I confirmed the update.